src/commands/updateservers.js

```diff
@@ -16,41 +16,56 @@ class UpdateserversCommand extends Command{
-        for(let i = 65; i < 91; i++){
-            const letter = String.fromCharCode(i);
-            const guildDocs = await guildModel.find({
-                name: {$regex: new RegExp(`^[^a-z]*${letter}`, 'i')},
-                pending: {$ne: true},
+        if ((new Date() - constants.lastGuildsChannelUpdate) < 600000) {
+            const confirmButton = new ButtonBuilder()
+                .setCustomId('collector:confirm')
+                .setLabel('Confirmar')
+                .setStyle(ButtonStyle.Success);
+            const cancelButton = new ButtonBuilder()
+                .setCustomId('collector:cancel')
+                .setLabel('Cancelar')
+                .setStyle(ButtonStyle.Danger);
+            
+            const reply = await interaction.editReply({
+                content: 'A última atualização foi a menos de 10 minutos, deseja atualizar novamente?',
+                components: [new ActionRowBuilder().setComponents(confirmButton, cancelButton)]
+            })
+
+            const collector = reply.createMessageComponentCollector({
+                time: 60000,
+                filter: (i) => i.user.id === interaction.user.id,
+                max: 1
             });
-            if(!guildDocs.length) continue;
-            guildCount += guildDocs.length;
-            const embed = new EmbedBuilder()
-                .setTitle(`Comunidades (${letter})`)
-                .setImage(`https://cdn.discordapp.com/attachments/946097024804212777/1037824425191542845/Divisoria.png`)
-                .setColor(0x5765f0)
-                .setDescription(
-                    guildDocs
-                        .map(doc => {
-                            let str = `[\`${doc.name.replaceAll('`', 'ˋ')}\`](https://discord.gg/${doc.invite})`;
-                            return doc.role ? `${str} | <@&${doc.role}>` : str;
-                        })
-                        .join('\n'),
-                );
-            await channel.send({embeds: [embed]});
+
+            collector.on('collect', async (i) => {
+                await i.deferUpdate()
+                if (i.customId === 'collector:cancel') {
+                    await interaction.editReply({ content: 'Operação cancelada', components: [] });
+                    return;
+                }
+                await interaction.editReply({ content: 'Lista de servidores atualizada', components: [] });
+                client.emit('updateGuilds', true);
+            })
+
+            collector.on('end', async (_, reason) => {
+                if (reason === 'time') {
+                    await interaction.deleteReply();
+                    await interaction.editReply({ content: 'Operação cancelada', components: [] });
+                    return;
+                }
+            })
+
+            return;
         }
-        await channel.setTopic(`<:icons_discover:859429432535023666>  | **${guildCount} servidores associados.**`);
+
+        client.emit('updateGuilds');
```

Comment:
explicacao no primeiro comentario em src/events/updateGuilds.js
```suggestion
        client.emit('updateGuilds', false);
```